### PR TITLE
Fix values

### PR DESCRIPTION
**Pull request by QuentinBisson**
Towards https://github.com/giantswarm/roadmap/issues/1547

### CODE REVIEW

**Review by mcbenjemaa — APPROVED** [review]

File: helm/prometheus-operator-app/values.yaml [hunk 1]
```diff
@@ -10,11 +10,17 @@ prometheus-operator-app:
...
+  coreDns:
+    service:
+      selector:
+        k8s-app: coredns
   grafana:
     ingress:
       ingressClassName: nginx
   kubeStateMetrics:
     enabled: false
+  nodeExporter:
```
Comment (hervenicol) [inline helm/prometheus-operator-app/values.yaml:22]: If you want to fix default enabled 3rdparties, nodeExporter is not the only one.
See https://github.com/giantswarm/config/pull/1332

**Review by hervenicol — CHANGES_REQUESTED** [review]
I'd prefer either you fix all default enabled 3rdparties, or none in this PR.

**Comment by QuentinBisson** [comment]
@hervenicol I did not fix kubeStateMetrics as it's already set to false and I did not remove the kubelet service because I think we should actually keep it enabled of workload clusters

**Review by hervenicol — APPROVED** [review]
Ok, fine with me then :)

**Pull request merged**